WordPressAuthenticator/Unified Auth/View Related/Get Started/GetStartedViewController.swift

```diff
@@ -313,7 +319,13 @@ private extension GetStartedViewController {
     // MARK: - Sign in with site credentials Button Action
     @objc func handleSiteCredentialsButtonTapped() {
         tracker.track(click: .signInWithSiteCredentials)
-        goToSiteCredentialsScreen()
+        guard WordPressAuthenticator.shared.configuration.checkXMLRPCOnlyIfSigningInUsingSiteCredentials else {
```

Comment:
Nit: this can be simplified as
```suggestion
        guard configuration.checkXMLRPCOnlyIfSigningInUsingSiteCredentials else {
```